### PR TITLE
feat-124:new feature

### PR DESCRIPTION
**Pull request by AssassinLYZ**
New feature is done

### CODE REVIEW

**Review by zezeindutch — COMMENTED** [review]
cool😎

**Review by zezeindutch — COMMENTED** [review]
cool

**Review by zezeindutch — APPROVED** [review]
good

**Review by zezeindutch — APPROVED** [review]
cool

File: customer/src/services/customer-service.test.js
Comment (zezeindutch) [inline customer/src/services/customer-service.test.js:1]: good

**Review by zezeindutch — APPROVED** [review]
ok

File: customer/src/services/customer-service.test.js
Comment (AssassinLYZ) [reply customer/src/services/customer-service.test.js:1]: ok

**Review by zezeindutch — APPROVED** [review]

File: customer/src/services/customer-service.test.js
Comment (zezeindutch) [reply customer/src/services/customer-service.test.js:1]: 啊啊啊

**Review by zezeindutch — COMMENTED** [review]
搜索

**Pull request merged**